### PR TITLE
Improve script for running scheduler benchmarks

### PR DESCRIPTION
**Pull request by wojtek-t**
Without this change, this script didn't work in my environment - it's making more consistent with other scripts.

@hongchaodeng @xiang90

### CODE REVIEW

File: test/component/scheduler/perf/test-performance.sh [hunk 1]
```diff
@@ -34,13 +37,10 @@ trap cleanup EXIT
 kube::etcd::start
 kube::log::status "performance test start"
 
-# TODO: set log-dir and prof output dir.
-DIR_BASENAME=$(basename `pwd`)
-go test -c -o "${DIR_BASENAME}.test"
 # We are using the benchmark suite to do profiling. Because it only runs a few pods and
 # theoretically it has less variance.
-"./${DIR_BASENAME}.test" -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -logtostderr=false
+go test -test.bench=. -test.run=xxxx -test.cpuprofile=prof.out -logtostderr=false
```
Comment (hongchaodeng) [inline test/component/scheduler/perf/test-performance.sh]: Profiling needs both binary and prof.out. Right?

Comment (wojtek-t) [inline test/component/scheduler/perf/test-performance.sh]: I'm not sure I understand the comment. I'm passing exactly the same parameters as before. And sure - we can pass more parameters, but that works too.

Comment (hongchaodeng) [inline test/component/scheduler/perf/test-performance.sh]: If we run normal benchmarking, we will do:
```
go test -run XXX -bench.
```

But if we want to do profiling, we need to
```
# compile it
go test -c # generate xxx.test
# benchmark and generate profile file
./xxx.test -test.run=xxx -test.bench=. -testcpuprofile=prof.out
# analyze profile output
go tool pprof xxx.test prof.out
```
The last step needs both binary and "prof.out".

Comment (gmarek) [inline test/component/scheduler/perf/test-performance.sh]: logtostderr=false is default - if nothing in binary overwrites it setting this flag is unnecessary.

**Comment by gmarek** [comment]
LGTM sans one nit.

File: test/component/scheduler/perf/test-performance.sh (hunk 1 above)
Comment (wojtek-t) [inline test/component/scheduler/perf/test-performance.sh]: OK - you mean not profiling itself (which works without having binary), but analyzing profile, right?

Comment (xiang90) [inline test/component/scheduler/perf/test-performance.sh]: @wojtek-t Yes. I believe he means so. Without the binary, we cannot analyze the output.

Comment (wojtek-t) [inline test/component/scheduler/perf/test-performance.sh]: done.

**Comment by wojtek-t** [comment]
@gmarek - comments applied; PTAL

**Comment by xiang90** [comment]
Another thing is that in the benchmark, we do not run spreading algo. Probably we need to fix that too to match with density test.

**Comment by xiang90** [comment]
LGTM

**Comment by wojtek-t** [comment]
@xiang90 - yes I wrote it in some bug.

Or to be more clear - you run the spreading priority function, but since there are no RC and Services, it just finishes immediately. Will send a fix tomorrow for it, since it seems there are some other things that are making the difference.

**Comment by xiang90** [comment]
> but since there are no RC and Services, it just finishes immediately.

Right.

**Pull request merged**